### PR TITLE
Add many methods

### PR DESCRIPTION
**Pull request by sharkAndshark**

### CODE REVIEW

File: LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs [hunk 1]
```diff
@@ -293,6 +296,102 @@ public void TestSTPoint()
             }
         }
 
+        [Test]
+        public void TestSTPointZ()
+        {
+            if (CurrentVersion < new Version("3.2.0")) return;
```
Comment (apdevelop) [inline LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs:302]: There are `TestsBase.VersionNNN` properties with versions definitions

File: LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs [hunk 2]
```diff
@@ -12,13 +12,16 @@ namespace LinqToDBPostGisNetTopologySuite.Tests
     [TestFixture]
     class GeometryConstructorsTests : TestsBase
     {
+        private Version CurrentVersion { get; set; }
```
Comment (apdevelop) [inline LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs]: I just added this field in most test classes in 9e429effb32e2190119c389b65d87011162163d2

Comment (sharkAndshark) [reply LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs]: Is it better pulling CurrentVersion up as field of TestBase?

Comment (apdevelop) [reply LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs]: Yes, getting current PostGIS version seems to be common code, may be moved to `TestsBase.OneTimeSetUp()`

Comment (sharkAndshark) [reply LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs]: In this commit or another?

Comment (apdevelop) [reply LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs]: Would be better in another commit

Comment (apdevelop) [reply LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs]: I will add them right now

**Pull request merged**